### ISSUE TITLE
menger_cubes example

### ISSUE THREAD

**Issue by smsaladi** [jewettaij/moltemplate#3]
I can't seem to run the menger_cubes example:

```text
moltemplate.sh v2.0.2 2016-12-04

lttree_check.py v0.76 2014-12-19

    "Data Atoms" column format:
    atom-ID molecule-ID atom-type q x y z

lttree_check.py:    parsing the class definitions...
Error(ttree.StaticObj.Parse()):
       Syntax error at or before "menger_cubes.lt", line 48:
       "cells_xz= new".
```

I looked at the syntax in `menger_cubes.lt` but can't seem to spot the error. Could you assist please?

**Comment by jewettaij** [comment]
You found an error in one of my favorite examples. (Dammit.)  Wow, great!

It's actually a nasty error, not in the menger_cubes example, but in the lexer ("ttree_lexer.lt")

I must have changed that example after I originally wrote it, and not bothered to test it.

The problem is the lack of spaces on either side of the '=' character.  This should not be necessary.  At some point I removed the '=' and ' and '*' characters from TtreeLex.word_terminators.  This was a bad idea, but the original intention was  to allow users to give their atoms type names like:
c=
O3'
c*
...which show up often in popular force-fields.
Don't worry about this.  I will fix it right now and communicate the changes
Thanks!

**Comment by jewettaij** [comment]
The bug was corrected release 2.0.5.  Thanks again Shyam
